adr_web_app/app.py

```diff
@@ -0,0 +1,38 @@
+from flask import Flask,render_template,Markup
+import os
+from adr.recipe import run_recipe
+import markdown
+
+app=Flask(__name__)
+recipes=[]
+
+def get_recipes():
+    """
+    Return a list of recipes located in /adr/recipes path
+    """
+    for root, dirs, files in os.walk("../adr/recipes"):

```

Comment:
Instead of hardcoding this path you can use:

    from adr import recipes
    recipe_path = os.path.dirname(recipes.__file__)

Reply:
Also instead of os.walk, you can just use `os.listdir`. You only need `os.walk` if you want to search a directory recursively, and we know that recipes don't live in subdirs.

Reply:
Thanks for comments, I will do it!